crypto/key_manager.go

```diff
@@ -25,7 +25,7 @@ import (
 	"io/ioutil"
 
 	"github.com/golang/glog"
-	"github.com/google/trillian"
+	spb "github.com/google/trillian/proto/signature"
```

Comment:
No strict conflicts.  I've had a habit of naming protobuf packages *pb for code clarity.